feat(planning_debug_tools): add stop reason visualizer

taikitanaka3 opened the pull request:
## Description

add stop reason visualizer in planning debug tools

![image](https://user-images.githubusercontent.com/65527974/189789178-123bd3e7-603b-475b-a11a-9f16b8ea0173.png)


## Related links

https://github.com/autowarefoundation/autoware.universe/issues/1845

## Tests performed

by psim

## Notes for reviewers

test this with below command and add to rviz marker
`ros2 run planning_debug_tools stop_reason_visualizer_exe`


## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

TakaHoribe reviewed (commented):
Great work! It would be nice if we can visualize the module name. It is now hard to see what logic is working for the stop point.

taikitanaka3 commented:
@TakaHoribe 
I added as a small text and added brief document
![image](https://user-images.githubusercontent.com/65527974/189809832-076d57b5-887a-4e5b-a65c-44478c841aab.png)

shmpwk reviewed (approved):
LGTM
